actions/transfer-case.vbs

```diff
@@ -138,466 +178,296 @@ EMReadScreen mail_addr_line_four, 43, 12, 27
+        call create_MAXIS_friendly_date(client_move_date, 0, 4, 28)    'Writing client move date
+        call create_MAXIS_friendly_date(client_move_date, 0, 4, 61)    'this is the CRF date we dont need to ask because we dont do this'
+        EMWriteScreen left(excluded_time_dropdown, 1), 5, 28            'Writes the excluded time info. Only need the left character (it's a dropdown)
+        IF excluded_time_dropdown = "Yes" THEN                          'If there's excluded time, need to write the info
+            call create_MAXIS_friendly_date(excluded_date, 0, 6, 28)
+            EMWriteScreen hc_cfr, 15, 39
+        END IF
+
+        IF excluded_date = "" AND excluded_time_dropdown = "No" THEN
+            EMWriteScreen "__", 6, 28
+            EMWriteScreen "__", 6, 31
+            EMWriteScreen "__", 6, 34
+        END IF
+
+        IF ma_status = "ACTIVE" AND manual_cfr_hc_checkbox = CHECKED THEN
+            EMWriteScreen hc_cfr, 14, 39
+            EMWriteScreen hc_cfr_month, 14, 53
+            EMWriteScreen hc_cfr_year, 14, 59
+        END IF
+
+        IF cash_status = "ACTIVE" AND manual_cfr_cash_checkbox = CHECKED THEN 'previously we read PROG for cash one and cash two programs unsure if this is necessary'
+            EMWriteScreen cash_cfr, 11, 39
+            EMWriteScreen cash_cfr_month, 11, 53
+            EMWriteScreen cash_cfr_year, 11, 59
+            EMReadScreen cash_cfr_two, 2, 12, 39
+            IF cash_cfr_two <> "__" THEN
+               EMWriteScreen cash_cfr, 12, 39
+               EMWriteScreen cash_cfr_month, 12, 53
+               EMWriteScreen cash_cfr_year, 12, 59
+            END IF
+        END IF
+
+        EMWriteScreen worker_number, 18, 28
+        EMWriteScreen transfer_to_worker, 18, 61
+        TRANSMIT                                           'saving - this should then take us to the transfer menu
+        EMReadScreen panel_check, 4, 2, 49                 'reading to see if we made it to the right place we shou.d be back on   Transfer Selection (XFER)
+        If panel_check <> "XFER" Then
+            transfer_case = False                       'this is not the right place
+            end_msg = "Transfer of this case to " & transfer_to_worker & " has failed."
+            PF10 'backout
+            PF3 'SPEC menu
+            PF3 'SELF Menu'
+        Else                                               'if we are in the right place - read to see if the new worker is the transfer_to_worker
+            EMReadScreen new_primary_worker, 7, 21, 16
+            IF new_primary_worker <> transfer_to_worker Then           'if it is not the transfer_to_worker - the transfer failed.
+                transfer_case = False
+                end_msg = "Transfer of this case to " & transfer_to_worker & " has failed."
+            End If
+        END IF
+    END IF ' confirming the xfer worked and we left the panel'
+END IF ' the big one'
+
+IF end_msg <> "" Then
```

Comment:
These if statements are a little redundant. If transfer case is false, there will always be information in 'end_msg' because they are defined in this way right next to each other. You might not need the 'transfer_case' boolean, or the if statement might be able to be just a little cleaner.

Reply:
removed transfer_case because it was a superfluous variable